Gemfile

```diff
@@ -53,6 +53,8 @@ gem 'turbolinks', '~> 2.5.3'
 
 gem 'uglifier', '>= 1.3.0'
 
+gem 'rails-i18n', '~> 4.0.0'
```

Comment:
Hi @jiteshjha 👋 

One minor suggestion, I think the Gems are ordered in alphabetical order (except for the `rails` gem). 

What do you think about inserting this line like this:

```diff
  gem 'rack-canonical-host'
  gem 'rack-timeout', require: false
+ gem 'rails-i18n', '~> 4.0.0'
  gem 'redis-namespace'
```

Thanks!